content/en/docs/collector/internal-telemetry.md

```diff
@@ -34,23 +36,26 @@ service:
       address: '0.0.0.0:8888'
 ```
 
-You can enhance the metrics telemetry level using the `level` field. The
-following is a list of all possible values and their explanations.
+You can adjust the verbosity of the Collector metrics output by setting the
+`level` field to one of the following values:
 
-- `none` indicates that no telemetry data should be collected.
-- `basic` is the recommended value and covers the basics of the service
-  telemetry.
-- `normal` adds other indicators on top of basic.
-- `detailed` adds dimensions and views to the previous levels.
+- `none`: no telemetry is collected.
+- `basic`: essential service telemetry.
+- `normal`: the default level, adds standard indicators on top of basic.
+- `detailed`: the most verbose level, includes dimensions and views.
```

Comment:
Hmm, I just noticed that the `level` field is also describe, in the same detail in http://localhost:1313/docs/collector/configuration/#telemetry:

> - `level`: the level of telemetry metrics, default `basic`. The possible values
>   are:
>   - "none" indicates that no telemetry data should be collected.
>   - "basic" is the recommended and covers the basics of the service telemetry.
>   - "normal" adds some other indicators on top of basic.
>   - "detailed" adds dimensions and views to the previous levels.

We should avoid the duplication of the full details.

Reply:
I will fix this in a follow-up PR. There is more on that page to incorporate here than just the `level` definitions. Enough to warrant a separate PR. Thanks!

Reply:
Put up an issue: #4476.